code/game/objects/items/cards_ids.dm

```diff
@@ -1700,11 +1700,11 @@
 		return
 
 	///forge the ID if not forged.
-	var/input_name = tgui_input_text(user, "What name would you like to put on this card? Leave blank to randomise.", "Agent card name", registered_name ? registered_name : (ishuman(user) ? user.real_name : user.name), MAX_NAME_LEN)
+	var/input_name = tgui_input_text(user, "What name would you like to put on this card? Leave blank to randomise.", "Agent card name", registered_name ? registered_name : (ishuman(user) ? user.real_name : user.name), MAX_NAME_LEN, encode = FALSE)
 	if(!after_input_check(user))
 		return TRUE
 	if(input_name)
-		input_name = sanitize_name(input_name, allow_numbers = TRUE)
```

Comment:
I'm prety sure this shouldn't be changed, switching to just reject_bad_name leaves out ic word filtering

Reply:
reject_bad_name is how pref names are also filtered. Sanitize_name() is overly aggressive.

Reply:
Most things taking direct player input use sanitize_name and reject_bad_name for prefs should probably also be switched out? I'm not sure why that's skipping past ic word filters.

Reply:
need to fix sanitize_name in that case, or at least figure an alternative solution

Reply:
Except more things use sanitize_name than reject_bad_name. This pr is *anti* consistency

Reply:
Removed that change as the issue behind it is non-pressing (doesn't expose card users or output broken characters into player chat) but is caused by byond's html sanitization so fixing it would be a pain